index.html

```diff
@@ -82,6 +85,37 @@ <h1>CODEVID-19</h1>
     </div>
   </section>
 
+  <!-- Calendar -->
+  <section id="calendar" class="wrapper style3 fade-up">
+    <div class="inner">
+      <h2>Calendar</h2>
+      <p>
+        <strong>Note:</strong> Times are in UTC,
+        <a
+          href="https://calendar.google.com/event?action=TEMPLATE&tmeid=NWY3b2E1MWNnamxrdm83am04b2VwanA3aHMgZGV2ZWRtb250b24uY29tX2Z0OG5vY2w1Y285Nzk1MHQ3a2ExcTRicm04QGc&tmsrc=devedmonton.com_ft8nocl5co97950t7ka1q4brm8%40group.calendar.google.com"
```

Comment:
this goes to an event, it needs to go to the codevid calendar

Reply:
Ah, sorry.  Fixed.